src/review/routes/homepage/Homepage.js

```diff
@@ -1,13 +1,23 @@
 import React, { PropTypes } from 'react'
 import { connect } from 'react-redux'
 import ProductHome from 'review/routes/product/Home'
+import Error from 'review/routes/Error'
 import Dashboard from 'review/routes/dashboard/Dashboard'
 
-function Homepage(props) {
-  return props.user ? <Dashboard /> : <ProductHome />
+function Homepage({ user, error }) {
```

Comment:
No it doesn't break it I had the warning. I think destructuring is more explicit and dryer.